http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -222,10 +222,12 @@ class MonitorAtBoot[F[_]](publisherQueue: Queue[F, LeoPubsubMessage],
                       appContext.traceId
                     )
                   )
-                  tokenOpt <- samDAO.getCachedArbitraryPetAccessToken(app.auditInfo.creator)
-                  workspaceDescOpt <- tokenOpt.flatTraverse { token =>
-                    wsmClientProvider.getWorkspace(token, workspaceId)
-                  }
+                  leoAuth <- samDAO.getLeoAuthToken
```

Comment:
Why the change here for both GCP and Azure case? I would like to keep using the cached token wherever possible.

Reply:
fixed